src/main/java/com/smartbear/readyapi/client/teststeps/datasource/datagen/PhoneNumberDataGeneratorBuilder.java

```diff
@@ -0,0 +1,33 @@
+package com.smartbear.readyapi.client.teststeps.datasource.datagen;
+
+import com.smartbear.readyapi.client.model.DataGenerator;
+import com.smartbear.readyapi.client.model.PhoneNumberDataGenerator;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class PhoneNumberDataGeneratorBuilder extends AbstractDataGeneratorBuilder<PhoneNumberDataGeneratorBuilder> {
+    private final PhoneNumberDataGenerator phoneNumberDataGenerator = new PhoneNumberDataGenerator();
+
+    private final String numberFormatString = "XXX-XXX-XXXX,+1 XXX-XXX-XXXX,+1 (XXX)-XXX-XXXX,+X XXX-XXX-XXXX,+X (XXX)-XXX-XXXX";
```

Comment:
Swagger codegen was not able to create enum for these values so I have to keep them in client code.